main.go

```diff
@@ -176,8 +177,12 @@ func main() {
 		}
 		logger.Fatal().Msgf("could not get ethereum address")
 	}
+
+	// get vehicle definitions from Identity API service
+	vehicleDefinition := getVehicleInfo(err, logger, ethAddr)
+
 	// OBD / CAN Loggers
-	ds := network.NewDataSender(unitID, *ethAddr, logger)
+	ds := network.NewDataSender(unitID, *ethAddr, logger, vehicleDefinition)
```

Comment:
if we were unable to get the vehicleDefinition, would this be nil? does the datasender handle that case...
maybe for future way to cache it on disk for future use if we're unable to get it from identity-api (network issue etc)

Reply:
There are two cases when `vehicleDefinition` can return as nil:
1.  due to a network issue 
2. the vehicle is not paired so that the response would be "not found"

Data sender would handle the nil and carry over tokenId, make, model, and year with null values.